core/src/main/java/org/apache/accumulo/core/client/AccumuloClient.java

```diff
@@ -52,7 +52,7 @@
  *
  * <p>
  * If migrating code from Connector to AccumuloClient an important difference to consider is that
- * AccumuloClient is closable and Connector is not. Connector uses static resources and therefore
+ * AccumuloClient is closable and Connector was not. Connector uses static resources and therefore
  * creating them is cheap. AccumuloClient attempts to clean up resources on close, so constantly
```

Comment:
The Connector used static resources and was cheaper to create that the AccumuloClient.  However, ...

Reply:
Addressed in 49dd2789c4